pom.xml

```diff
@@ -1474,7 +1474,7 @@
             <dependency>
                 <groupId>com.github.luben</groupId>
                 <artifactId>zstd-jni</artifactId>
-                <version>1.3.5-4</version>
+                <version>1.4.4-2</version>
```

Comment:
I believe the latest version is 1.4.5-4 https://github.com/luben/zstd-jni/releases/tag/v1.4.5-4. Can we upgrade to that?

Reply:
The reason I used 1.4.4-2 as its the latest available in the FB repo.

Reply:
I will switch to the latest version available in the open source.